src/components/App.jsx

```diff
@@ -33,6 +34,8 @@ import About from './pages/About';
 import Alert from 'react-popup';
 import 'react-popup/style.css';
 
+import { setLoading as reduxAction_setLoading } from '../redux/actions';
```

Comment:
This isn't a great name, and `reduxAction_setLoading` is even worse. How about `setAppLoading`?

Reply:
Yeah probably better